twinify/napsu_mq/napsu_mq.py

```diff
@@ -0,0 +1,186 @@
+# Copyright 2022 twinify Developers and their Assignees
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from typing import Optional, Union, Iterable, BinaryIO, Callable, Mapping
+import os
+import pandas as pd
+
+import numpy as np
+import arviz as az
+import jax
+import jax.numpy as jnp
+from mbi import Domain, Dataset
+import dill
+
+from twinify.napsu_mq.mst import MST_selection
+from twinify.base import InferenceModel, InferenceResult, InvalidFileFormatException
+from twinify.napsu_mq.markov_network_jax import MarkovNetworkJax
+from twinify.napsu_mq.marginal_query import FullMarginalQuerySet
+from twinify.napsu_mq.dataframe_data import DataFrameData
+import twinify.napsu_mq.privacy_accounting as privacy_accounting
+import twinify.napsu_mq.maximum_entropy_inference as mei
+
+from d3p.random import PRNGState
+
+
+class NapsuMQModel(InferenceModel):
+
+    def __init__(self) -> None:
+        super().__init__()
+
+    def fit(self, data: pd.DataFrame, rng: PRNGState, epsilon: float, delta: float, **kwargs) -> 'NapsuMQResult':
```

Comment:
`column_feature_sets` and `use_laplace_approximation` should be arguments to `__init__`, as they are properties of the model like Lukas said.